code/modules/antagonists/heretic/knowledge/side_rust_cosmos.dm

```diff
@@ -18,6 +18,26 @@
 	cost = 1
 	route = PATH_SIDE
 
+/datum/heretic_knowledge/entropy_pulse
+	name = "Pulse of Entropy"
+	desc = "Allows you to transmute 20 irons and 2 garbage items to fill the surrounding vicinity of the rune with rust."
+	gain_text = "Reality begins to whisper to me. To give it its entropic end."
+	required_atoms = list(
+		/obj/item/stack/sheet/iron = 20,
+		/obj/item/trash = 2
+	)
+	cost = 1
+	route = PATH_SIDE
+	var/size = 49 ///roughly 3 tiles radius
```

Comment:
There is no way the maints are going to allow this weird-ass way to do this man.